runtime/auth/http-auth/common/src/aws/smithy/kotlin/runtime/auth/HttpSigner.kt

```diff
@@ -3,11 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package aws.smithy.kotlin.runtime.http.auth
+package aws.smithy.kotlin.runtime.auth
```

Comment:
**Question**: I'd expect the package name to follow the module name. Since this is the **http-auth** module, why put this class in the `aws.smithy.kotlin.runtime.auth` package (instead of `aws.smithy.kotlin.runtime.http.auth`)?